pallets/gear-voucher/src/mock.rs

```diff
@@ -117,6 +141,7 @@ impl pallet_gear_voucher::Config for Test {
     type WeightInfo = ();
     type CallsDispatcher = ();
```

Comment:
It will require a lot of other thing to be implemented in the mock. Can't see any reason for that cause this pallet tests vouchers logic totally except processing of the messages it sends, but we have those tests in pallet gear (where processing is implemented)